saylua/templates/navbar.html

```diff
@@ -97,7 +97,7 @@
       </a>
       <div id="dropdown-user-menu" class="dropdown-menu">
         <div id="user-menu" class="menu">
-          <h3>Logged in as {{g.user.display_name}}</h3>
+          <h3>Logged in as {{g.user.display_name.display_name}}</h3>
```

Comment:
We've really got to change this before it's too far gone, haha. `display_name.name` or `display_name.username` would be way better.

This is unrelated to your changes, don't feel obligated to fix this.